framework/console/controllers/MigrateController.php

```diff
@@ -293,8 +295,8 @@ protected function generateMigrationSourceCode($params)
 
             $foreignKeys[$firstTable . '_id']['table'] = $firstTable;
             $foreignKeys[$secondTable . '_id']['table'] = $secondTable;
-            $foreignKeys[$firstTable . '_id']['column'] = 'id';
-            $foreignKeys[$secondTable . '_id']['column'] = 'id';
+            $foreignKeys[$firstTable . '_id']['column'] = '';
```

Comment:
@samdark  if here use just ```id```, then here https://github.com/MKiselev/yii2/blob/4126e46c3889373a54d59a22fe49c5b485d370c1/framework/console/controllers/MigrateController.php#L320 check will be false.